cmd/cluster-kube-apiserver-operator/render/render.go

```diff
@@ -0,0 +1,100 @@
+package render
+
+import (
+	"errors"
+
+	"github.com/golang/glog"
+	"github.com/openshift/cluster-kube-apiserver-operator/pkg/assets"
+	"github.com/spf13/cobra"
+)
+
+type renderOpts struct {
+	assetDir             string
+	secretsDir           string
+	manifestTemplatesDir string
+	configDir            string
+	dataDir              string
+	cloudProviderDir     string
+}
+
+func NewRenderCommand() *cobra.Command {
+	renderOpts := &renderOpts{}
+	cmd := &cobra.Command{
+		Use:   "render",
+		Short: "Render kubernetes API server bootstrap manifests",
+		Run: func(cmd *cobra.Command, args []string) {
+			if err := renderOpts.Validate(); err != nil {
+				glog.Fatal(err)
+			}
+			if err := renderOpts.Run(); err != nil {
+				glog.Fatal(err)
+			}
+		},
+	}
+
+	cmd.Flags().StringVar(&renderOpts.assetDir, "asset-dir", "", "Output path for rendered assets.")
+	cmd.Flags().StringVar(&renderOpts.secretsDir, "secrets-input-dir", "", "A path to directory with certificates and secrets.")
```

Comment:
"secret" is a fixed term. Wouldn't use this here, especially if it contains certs as well which are not secret. How about `--assets-input-dir` ?

Reply:
in bootkube, "assets" are the manifests we generate out from render, if we change it here, it might be confusing for people that were familiar with bootkube (if we care)...